stylus/ui-app/modals.styl

```diff
@@ -0,0 +1,63 @@
+/*!
+ * NOTIFICATIONS
+ * =============
+ *
+ * This file contains all needs for modals
+ */
+
+
+$modals
+    modal-width = em(480px)
+    .coz-modal
+        z-index 100
+
+    .coz-overlay
+        position   absolute
+        height     100%
+        width      100%
+        top        0
+        left       0
+        background rgba(grey-08, .5)
+        visibility visible
+        transition opacity .3s, visibility 0s ease-out
+
+        &.coz-overlay--hidden
+            opacity          0
+            visibility       hidden
+            transition       opacity .3s 0s, visibility 0s ease-in .3s
+
+
+    .coz-modal
+        display          block
```

Comment:
Maybe a tiny change of order here. I'd say like this: 
1. Positionning (position, top, left), box-model (display, box-sizing, width, height), padding, etc